collector/cpu_openbsd_amd64.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !nocpu
+
+package collector
+
+import (
+	"strconv"
+	"unsafe"
+
+	"github.com/go-kit/kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"golang.org/x/sys/unix"
+)
+
+type clockinfo struct {
+	hz      int32
+	tick    int32
+	tickadj int32
+	stathz  int32
+	profhz  int32
+}
+
+const (
+	CP_USER   = 0
```

Comment:
Let's use https://golang.org/ref/spec#Iota here. Should be:
```
const (
    CP_USER = iota
    CP_NICE
    ...
)
```

Reply:
Done, plus a bit more code cleanup